sqlglot/dialects/mysql.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 from sqlglot import exp, generator, parser, tokens
+from sqlglot.dialects.bigquery import date_add
```

Comment:
At first I thought about drying this up by moving it in `dialect.py`, but we already have a `parse_date_delta` function which seems to do something similar, so I wasn't sure what name to use and left it like this.

Reply:
you should dry it up

Reply:
Any suggestion for the name? This is used for `DateSub` as well, so ideally I'd use something like `parse_date_delta`, but as I said that's already reserved. The easy way out is to move it to `dialect.py` as `date_add` and move on.

Reply:
Done

Reply:
Hadn't heard "dry it up" before, nice

Reply:
We use it frequently around here :-) See also: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
Yeah, got the DRY reference, just hadn't heard it verbed before 😄